test/box/gh-6292-fix-dropping-msg-when-connection-closing.test.lua

```diff
@@ -0,0 +1,58 @@
+net = require('net.box')
+fiber = require('fiber')
+test_run = require('test_run').new()
+
+box.schema.user.grant('guest', 'execute', 'universe')
+old_readahead = box.cfg.readahead
+old_net_msg_max = box.cfg.net_msg_max
+
+--
+-- Check that tarantool process all requests when connection is closing,
+-- net_msg_max limit is reached and readahead limit is not reached:
+--
+box.cfg ({ net_msg_max = 2, readahead = 4 * old_readahead })
```

Comment:
Better not depend on old_readahead here, because we don't know its value. Just set it to a very big value (say 100 MB).

Reply:
Fixed